### ISSUE TITLE
Optionally show the values of propagated parameters in the GUI

### ISSUE THREAD

**Issue by mpradovelasco** [OpenModelica/OpenModelica#11269]
Hi, 

The parameter N, which is defined in the base class `void` in the following model, is not considered by annotation clause when it is modified during the inheritance. 

``` Modelica
package GraphicText
  class void
    parameter Integer N = 1;
  end void;
  class M
    extends void(N = Nin);
    parameter Integer Nin = 2;
    annotation(
      Icon(graphics = {Text(origin = {4, 7}, extent = {{-34, 29}, {34, -29}}, textString = "%N")}));
  end M;
  model View
    M M1 
    annotation(
        Placement(visible = true, transformation(origin = {-4, 2}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
  end View;
end GraphicText;
```
That is, the M1 component shows Nin instead of 2.
![image](https://github.com/OpenModelica/OpenModelica/assets/114144098/d08bdd08-8853-4f2f-857a-26c2f7688d3b)

However, annotation works fine when 2 is applied directly as modifier, that is
``` Modelica
class M
  extends void(N = 2);
  parameter Integer Nin = 2;
  annotation(
    Icon(graphics = {Text(origin = {4, 7}, extent = {{-34, 29}, {34, -29}}, textString = "%N")}));
end M;
```
with the following diagram view:
![image](https://github.com/OpenModelica/OpenModelica/assets/114144098/dfb4c49e-7b35-4cea-891e-fd7079d47628)

System: OMEdit 1.23-dev (23/9) on Windows 11. 

Thank you

**Comment by AustraleanPerson** [comment]
This is expected behaviour. Dymola works the same way.

**Comment by casella** [comment]
See the discussion in #8966. In case of parameter propagation, it is really a matter of taste whether you want to see the actual value or rather the name of the propagated parameter. @mpradovelasco, if this is useful for you, we could add a flag to force evaluating those parameters. What do you think?

**Comment by mpradovelasco** [comment]
Hi @casella . I have read the discussion in #8996 carefully and I do really consider that it is more useful to show the propagated parameter. This is very useful when the GUI is used by a model builder, that may not be interested to know that the value emerges after a propagation related to the internal design of the library. 

If you could add a flag to force evaluating those parameters I will be deeply happy! I am very near to deliver a first version of the open source CybSim, prepared to be used by pharmacometrics and bioengineering professionals, which uses extensively propagated attributes in a set of relevant components.

Thanks!

**Comment by casella** [comment]
@adeas31 one more feature to add to your to-do list. @mpradovelasco this may take some time, we have a bit of backlog with [OMEdit tickets](https://github.com/OpenModelica/OpenModelica/issues?q=label%3ACOMP%2FGUI%2FOMEdit+is%3Aopen).

**Comment by mpradovelasco** [comment]
I understand. You are doing a huge and fantastic work. Thanks again!

**Comment by casella** [comment]
When you publish CybSim it would be nice to have some more info (papers, documentation, website)

**Comment by AustraleanPerson** [comment]
From my understanding this is the correct variant:
```modelica
package GraphicText
  class void
    parameter Integer N = 1;
  end void;

  class M
    extends void(N = Nin);
    parameter Integer Nin = 2;
    annotation (
      Icon(graphics={  Text(origin = {4, 7}, extent = {{-34, 29}, {34, -29}}, textString = String(N))}));
  end M;

  model View
    M M1
    annotation (
        Placement(visible = true, transformation(origin = {-4, 2}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
  end View;
end GraphicText;
```
We use this approach in other modelica tool. @MaksKBV promised to fix string evaluation in OM. It's actually just one line of code. But several related issues were discovered, so the pull request is delayed.

**Comment by casella** [comment]
@mpradovelasco please check if this issue was resolved by #11844.

**Comment by mpradovelasco** [comment]
@casella, you all are great. The fix solves perfectly the issue. 

Thank you very much for your work!
